misk/src/main/kotlin/misk/web/WideOpenDevelopmentInterceptor.kt

```diff
@@ -0,0 +1,29 @@
+package misk.web
+
+import misk.Action
+import misk.environment.Environment
+import javax.inject.Inject
+
+internal class WideOpenDevelopmentInterceptor : NetworkInterceptor {
```

Comment:
document goals of this? Maybe TODO to remove?
(or should we remove it immediately?)

Reply:
Might leave it until I've tested with Webpack more but likely can be removed. I've added #278